ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -88,6 +90,21 @@ class PhaserUnit extends Phaser.GameObjects.Container {
 				label.visible = false;
 			});
 
+		this.fadingTextListener =
+			unit.on('fading-text', (config: {
+				text: string;
+				color?: string;
+			}) => {
+				console.log('PhaserUnit fading-text', unit.id()); // TODO remove
+				const data = {
+					text : config.text || '',
```

Comment:
same as above comment. I don't see why we have default values for `text` and `color` in this when we're making sure we always provide a value earlier.